app/grandchallenge/verifications/models.py

```diff
@@ -69,6 +70,16 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+    def user_info(self):
```

Comment:
This is a property of the model, so it's best to add `@property` to this method

Reply:
Ok, why is this preferred? I understand that adding `@property` makes `user_info` a property of the model and so we can then call `verification.user_info` rather than `verification.user_info()`, but I don't reallyunderstand why the former is preferred over the latter. Is it faster, more efficient?